wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/PruningRules.kt

```diff
@@ -226,7 +233,18 @@ class PruningRules private constructor(builder: Builder) {
       this.until = until?.toLowerCaseSemVer()
     }
 
+    /**
+     * The only version of the version range. Fields with `until` values greater than this, as well
+     * as fields with `since` values less than or equal to this, are retained.
+     */
+    fun only(only: String?) = apply {
+      this.only = only?.toLowerCaseSemVer()
+    }
+
     fun build(): PruningRules {
+      check(only == null || since == null && until == null) {
```

Comment:
Would prefer clarifying parenthesis to clearly denote precedence

Reply:
fixed